library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Generalized-photoparoxysmal-intermittent-photic-stimulation-response-limited <nowiki> {suggestedTag=Finding-frequency} [Limited to the stimulus train: Comprises multiple spikes or spike-and-wave activity, which are apparently generalized, but may be of greater amplitude at the front or back of the head. It is termed a photoconvulsive response (PCR) by Bickford et al., and corresponds to type 4 response of Waltz et al. .]</nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Generalized-photoparoxysmal-intermittent-photic-stimulation-response-self-sustained <nowiki> {suggestedTag=Finding-frequency} [Self-sustaining: Generalized PPR continuing after stimulation. This may not be demonstrated unless the stimulus train is terminated as soon as a generalized PPR is identified. It was termed prolonged photoconvulsive response by Reilly and Peters, and has a strong association with epilepsy and visually induced seizures in patients referred for clinical EEG examination. Its prevalence in asymptomatic general populations is unknown, but was found in five of 13,658 apparently healthy aircrew by Gregory et al..]</nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Activation-of-pre-existing-epileptogenic-area-intermittent-photic-stimulation-effect <nowiki> {suggestedTag=Finding-frequency} [Rarely, photic stimulation may activate an epileptogenic cortex, which is also spontaneously active; IPS could then also elicit a seizure by stimulating this, usually posterior located, area. It is questionable whether this should be considered a PPR, and it does not figure in established classifications.]</nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Unmodified-intermittent-photic-stimulation-effect
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
 
-        ** Quality-of-hyperventilation 
-            *** Hyperventilation-refused-procedure
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Hyperventilation-poor-effort
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Hyperventilation-good-effort
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Hyperventilation-excellent-effort
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+        ** Intermittent-photic-stimulation-effect <nowiki>{suggestedTag=Normal}[The effect of Intermittent Photic Stimulation (IPS) is scored according to the terminology proposed by Kasteleijn-Nolst Trenité ea (2001). (Source: Beniczky ea 2017.)]</nowiki>
```

Comment:
We might want to have a discussion about this section --- the tag names are very long -- it might be possible to organize them into subcategories or use orthogonality.

Reply:
stick to paper definition, take out IPS term